backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/switch/BpkSwitch.kt

```diff
@@ -54,24 +58,64 @@ fun BpkSwitch(
     onCheckedChange: ((Boolean) -> Unit)?,
     modifier: Modifier = Modifier,
     enabled: Boolean = true,
+    shouldTruncate: Boolean = false,
```

Comment:
QQ: 
I can see in the slack conversation, `shouldTruncate` should be set to `true` per default.
any reason why the default here is false, same for the ones bellow